src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,120 @@
 import React from 'react';
 import './TodoList.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.Component {
+  state = {
+    taskToFind: '',
+    selectedTodos: 'all',
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
+
+  render() {
+    const { todos, selectUser, selectedUserId } = this.props;
+    const { taskToFind, selectedTodos } = this.state;
+
+    const findTodos = todos.filter(todo => (
+      todo.title !== null
+      && todo.title.toLowerCase().includes(taskToFind.toLowerCase())
+    ));
+
+    const filterTodos = findTodos.filter((todo) => {
```

Comment:
For the future - it's better to make all filtering at the top level and pass already prepared todos to the 'TodoList' component